dags/atd-parking-data.py

```diff
@@ -0,0 +1,263 @@
+        "opitem": "Parking Data ETL",
+        "opfield": "socrata.Fiserv Payments Dataset",
+    },
+    "METERS_DATASET": {
+        "opitem": "Parking Data ETL",
+        "opfield": "socrata.Parking Meters Dataset",
+    },
+    "PAYMENTS_DATASET": {
+        "opitem": "Parking Data ETL",
+        "opfield": "socrata.Parking Meter Credit Card Payments Dataset",
+    },
+    # PostgREST
+    "POSTGREST_TOKEN": {
+        "opitem": "Parking Data ETL",
+        "opfield": "postgrest.Token",
+    },
+    "POSTGREST_ENDPOINT": {
+        "opitem": "Parking Data ETL",
+        "opfield": "postgrest.Endpoint",
+    },
+    # Passport
+    "OPS_MAN_USER": {
+        "opitem": "Parking Data ETL",
+        "opfield": "passport.Username",
+    },
+    "OPS_MAN_PASS": {
+        "opitem": "Parking Data ETL",
+        "opfield": "passport.Password",
+    },
+    # Fiserv
+    "FSRV_EMAIL": {
+        "opitem": "Parking Data ETL",
+        "opfield": "fiserv.Expected Email Address",
+    },
+    # AWS S3
+    "AWS_ACCESS_ID": {
+        "opitem": "Parking Data ETL",
+        "opfield": "aws.Access ID",
+    },
+    "AWS_ACCESS_ID": {
+        "opitem": "Parking Data ETL",
+        "opfield": "aws.Access ID",
+    },
+    "AWS_ACCESS_KEY_ID": {
+        "opitem": "Parking Data ETL",
+        "opfield": "aws.Access ID",
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+        "opitem": "Parking Data ETL",
+        "opfield": "aws.Secret Access Key",
+    },
+    "AWS_PASS": {
+        "opitem": "Parking Data ETL",
+        "opfield": "aws.Secret Access Key",
+    },
+    "BUCKET": {
+        "opitem": "Parking Data ETL",
+        "opfield": "aws.Bucket Name",
+    },
+    "BUCKET_NAME": {
+        "opitem": "Parking Data ETL",
+        "opfield": "aws.Bucket Name",
+    },
+    # Flowbird
+    "ENDPOINT": {
+        "opitem": "Parking Data ETL",
+        "opfield": "flowbird.Endpoint",
+    },
+    "USER": {
+        "opitem": "Parking Data ETL",
+        "opfield": "flowbird.ATD Username",
+    },
+    "PASSWORD": {
+        "opitem": "Parking Data ETL",
+        "opfield": "flowbird.ATD Password",
+    },
+    "USER_PARD": {
+        "opitem": "Parking Data ETL",
+        "opfield": "flowbird.PARD Username",
+    },
+    "PASSWORD_PARD": {
+        "opitem": "Parking Data ETL",
+        "opfield": "flowbird.PARD Password",
+    },
+}
+
+docker_commands = [
+    "python txn_history.py -v --report transactions",
+    "python txn_history.py -v --report payments",
+    "python txn_history.py -v --report payments --user pard",
+    "python passport_txns.py -v",
+    "python fiserv_email_pub.py",
+    "python fiserv_DB.py",
+    "python payments_s3.py",
+    "python payments_s3.py --user pard",
+    "python passport_DB.py",
+    "python smartfolio_s3.py",
+    "python match_field_processing.py",
+    "python parking_socrata.py --dataset payments",
+    "python parking_socrata.py --dataset fiserv",
+    "python parking_socrata.py --dataset transactions",
+]
```

Comment:
List of docker commands that are run. Was hoping to be able to parameterize this but ultimately got frustrated with the [docs](https://airflow.apache.org/docs/apache-airflow/2.6.0/core-concepts/params.html) gave up